### PR TITLE
Added Webhook mode,

### PR DESCRIPTION
**Pull request by amedranogil**
Added a flask endpoint, so the container can be executed as a daemon and called from Sonnar when configured as webhook.
This is optimal for containerised deployments as there is effectively 0 configuration on the host.

### CODE REVIEW

**Comment by jonfairbanks** [comment]
Thanks for opening this PR. This was my eventual plan so thanks for the help! 😄

File: Dockerfile [hunk 1]
```diff
@@ -2,18 +2,16 @@ FROM python:2.7-alpine
 
 RUN pip install --upgrade pip
 
-RUN apk add --no-cache tini
-
 RUN adduser -D python
 RUN mkdir /home/python/app/ && chown -R python:python /home/python/app
 WORKDIR /home/python/app
 
-ENTRYPOINT ["/sbin/tini", "--"]
```
Comment (jonfairbanks) [inline Dockerfile:11]: Any particular reason for removing tini?

File: README.md [hunk 2]
```diff
@@ -6,14 +6,32 @@ Updated fork of [https://gitlab.com/spoatacus/sonarr-episode-trimmer](https://gi
...
+    `docker run --rm --name=sonarr-trimmer -e URL=http://myurl -e API_KEY=1234 jonfairbanks/sonarr-episode-trimmer --list-series`
+
+### Automate
 To automate this clean-up, create a similar entry in cron with `crontab -e`:
 ```
 0 * * * * docker run --rm --name sonarr-trimmer jonfairbanks/sonarr-episode-trimmer
-```
\ No newline at end of file
+```
+### Webhook mode
```
Comment (jonfairbanks) [inline README.md:32]: Thank you for docs! ❤️

File: Dockerfile (hunk 1 above)
Comment (amedranogil) [reply Dockerfile:11]: I think in a container as simple as this one, tini is not needed. 
reduces footprint, complexity, and the new entrypoint is more standard as a [CLI package](https://andrewlock.net/packaging-cli-programs-into-docker-images-to-avoid-dependency-hell/)

**Pull request merged**